frontend/workflows/k8s/src/resize-hpa.tsx

```diff
@@ -74,6 +74,7 @@ const HPADetails: React.FC<WizardChild> = () => {
           {
             name: "Min Size",
             value: hpa.sizing.minReplicas,
+            disabledFieldlabel: "Current size",
```

Comment:
we have a few workflows that have mutable rows on the metadata table so thought it'd be better to make the label for the disabled text field be prop based so that workflows can customize it/enable it (rather than setting `Current` by default) . lmk if you have other preferences for the prop name